kratos/tests/test_linear_constraints.py

```diff
@@ -0,0 +1,194 @@
+from __future__ import print_function, absolute_import, division
+import KratosMultiphysics
+import KratosMultiphysics.StructuralMechanicsApplication as StructuralMechanicsApplication
```

Comment:
this has to be protected, otherwise crashes when Structural is not compiled
```suggestion
```

Reply:
yes i wanted to do that but i didn't remove it from above